src/data-table/types.ts

```diff
@@ -0,0 +1,73 @@
+import { CSSProperties, ReactNode } from "react";
+
+export interface DataTableProps {
+    testid?: string | undefined;
+    headersConfig?: HeadersConfigProps | undefined;
+    headers: HeadersProps[];
+    rowsConfig?: RowsConfigProps | undefined;
+    rows?: RowProps[] | undefined;
+    className?: string | undefined; // css class to put on table
```

Comment:
set these comments as jsdocs instead
```suggestion
    /** css class to put on table **/
    className?: string | undefined;
```